hardhat.config.ts

```diff
@@ -24,13 +41,14 @@ module.exports = {
   namedAccounts: {
     deployer: {
       default: 0,
-    },
-    // Temp to get some deployments working
-    manager: {
-      default: 2,
+      alfajores: ALFAJORES_FROM,
+      staging: STAGING_FROM,
```

Comment:
These you run deployment with a --from, the deploy task will override these with the provided address.. 
e.g

yarn deploy:alfajores --url "http://localhost:8545"  --from $CELO_ACCOUNT_ADDRESS

in this case  `ALFAJORES_FROM` will be changed at deploy time to `$CELO_ACCOUNT_ADDRESS`